test/handler/test_flow_handler.vader

```diff
@@ -1,4 +1,15 @@
 Before:
+  runtime ale_linters/javascript/flow.vim
+
+After:
+  unlet! g:flow_output
+  unlet! g:expected
+  unlet! g:actual
+  call ale#linter#Reset()
+
+Execute(The flow handler should process errors correctly.):
+  e! /home/w0rp/Downloads/graphql-js/src/language/parser.js
+
   let g:flow_output = {
```

Comment:
Inspired by the ESLint handler tests, I moved the output from the `After` block into the `Execute` block.